Sources/Danger/BitBucketServerDSL.swift

```diff
@@ -19,401 +17,382 @@ public struct BitBucketServer: Decodable, Equatable {
 
-    /// When the commit was commited to the project
-    public let committerTimestamp: Int?
+                /// The state of the task (e.g. "OPEN")
+                public let state: String
 
-    /// The commit's message
-    public let message: String
+                /// The author of the comment
+                public let author: User
+            }
 
-    /// The commit's parents
-    public let parents: [BitBucketServerCommitParent]
+            public struct InnerProperties: Decodable, Equatable {
+                /// The ID of the repo
+                public let repositoryId: Int
 
-    // MARK: - BitBucketServerCommitParent
+                /// Slugs of linkd Jira issues
+                public let issues: [String]?
+            }
+        }
+    }
+}
 
-    public struct BitBucketServerCommitParent: Decodable, Equatable {
+extension BitBucketServer {
+    public struct Commit: Decodable, Equatable {
         /// The SHA for the commit
         public let id: String
 
         /// The shortened SHA for the commit
         public let displayId: String
-    }
-}
 
-// MARK: - BitBucketServerPR
+        /// The author of the commit, assumed to be the person who wrote the code.
+        public let author: User
 
-public struct BitBucketServerPR: Decodable, Equatable {
-    enum CodingKeys: String, CodingKey {
-        case id
-        case version
-        case title
-        case description
-        case state
-        case open
-        case closed
-        case createdAt = "createdDate"
-        case updatedAt = "updatedDate"
-        case fromRef
-        case toRef
-        case isLocked = "locked"
-        case author
-        case reviewers
-        case participants
-    }
+        /// The UNIX timestamp for when the commit was authored
+        public let authorTimestamp: Int
 
-    /// The PR's ID
-    public let id: Int
+        /// The author of the commit, assumed to be the person who commited/merged the code into a project.
+        public let committer: User?
 
-    /// The API version
-    public let version: Int
+        /// When the commit was commited to the project
+        public let committerTimestamp: Int?
 
-    /// Title of the pull request.
-    public let title: String
+        /// The commit's message
+        public let message: String
 
-    /// The description of the PR
-    public let description: String?
+        /// The commit's parents
+        public let parents: [Parent]
 
-    /// The pull request's current status.
-    public let state: String
+        public struct Parent: Decodable, Equatable {
+            /// The SHA for the commit
+            public let id: String
 
-    /// Is PR open?
-    public let open: Bool
+            /// The shortened SHA for the commit
+            public let displayId: String
+        }
+    }
+}
 
-    /// Is PR closed?
-    public let closed: Bool
+extension BitBucketServer {
+    public struct PR: Decodable, Equatable {
```

Comment:
<!--
  0 failure: 
  1 warning:  Type name should ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketServerDSL.swift;
  Line: 217;
-->

- :warning: Type name should be between 3 and 40 characters long: 'PR' (`type_name`)